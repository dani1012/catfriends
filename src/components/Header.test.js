import React from 'react';
import Header from './Header';
import Enzyme from 'enzyme';
import {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';


Enzyme.configure({ adapter: new Adapter() });



it ('expects to render Header component', () => {
	const wrapper = shallow(<Header/>);
	expect(toJson(wrapper)).toMatchSnapshot();
})