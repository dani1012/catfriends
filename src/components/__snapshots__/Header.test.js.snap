// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render Header component 1`] = `
<div>
  <h1
    className="f1"
  >
    CATFRIENDS
  </h1>
  <CounterButton />
</div>
`;
